node/src/main/scala/coop/rchain/node/main.scala

```diff
@@ -111,6 +111,15 @@ object Main {
       def generateNonce: Task[Nonce] = Task.delay {
         encoder.decode("69696ee955b62b73cd62bda875fc73d68219e0036b7a0b37")
       }
+
+      def encrypt(pub: Key, sec: Key, nonce: Nonce, message: Array[Byte]): Task[Array[Byte]] =
+        Task.delay {
+          message
```

Comment:
What's going on here?

Reply:
`//TODO FIX-ME temp implemmentation, production shoul use crypto module` should tell ou everything :)

This is mock implementation. Adding real encryption is part of next PR https://github.com/rchain/rchain/pull/410/files#diff-99c215e4c2e53bf45b4887028939a176L100

Reply:
gotcha